iceoryx2-bb/elementary/tests/package_version_tests.rs

```diff
@@ -15,6 +15,13 @@ use iceoryx2_bb_testing::assert_that;
 
 #[test]
 fn package_version_works() {
+    // NOTE: The test is skipped when not run with cargo but with bazel
+    //       The CI which runs with cargo ensures that the constants defined
+    //       in PackageVersion::get equal the package version.
+    if option_env!("CARGO").is_none() {
```

Comment:
In `iceoryx2_bb_testing` there is a macro called `test_requires` which does this little thing for you here.

Could you please rewrite this to

```rust
fn package_version_works() {
  test_requires!(option_env!("CARGO").is_some());
  // ...
```